lib/executor/vu_handle.go

```diff
@@ -154,13 +155,11 @@ mainLoop:
 			if err != nil {
 				return
 			}
-			deactivateVU = func() {
-				vh.returnVU(initVU)
-			}
-			vu = initVU.Activate(&lib.VUActivationParams{
-				RunContext:         ctx,
-				DeactivateCallback: deactivateVU,
-			})
+			activationParams := getVUActivationParams(ctx, *vh.config,
+				func(u lib.InitializedVU) {
+					vh.returnVU(u)
+				})
```

Comment:
nit: because `vh.returnVU` has the same signature you should be able to just say `getVUActivationParams(ctx, *vh.config, vh.returnVU)`

Reply:
Indeed, thanks. Resolved by 56c0a891.